tomls/omnibus-base-sepolia-andromeda/spot/cbbtc.toml

```
@@ -0,0 +1,92 @@
+[var.cbbtc_spot_settings]
+synth_cbbtc_max_collateral_amount = "<%= parseEther(String(100)) %>"
+synth_cbbtc_skew_scale = "<%= parseEther(String(35_000)) %>"
+
+[invoke.SpotMarketProxy_createSynth_cbbtc]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic Coinbase Bitcoin", "sCBBTC", "<%= settings.owner %>"]
```

```suggestion
args = ["Synthetic Coinbase Wrapped BTC", "scbBTC", "<%= settings.owner %>"]
```
from https://help.coinbase.com/en/coinbase/trading-and-funding/sending-or-receiving-cryptocurrency/coinbase-wrapped-btc

---

yeah this is often confusing the casing needs to be right, i agree, but with an `s` there makes these decisions a bit more complicated tbh, so i would just like to uniformize these, in a way, where the actual asset in upper case and the protocol/cex it comes from in lower... what do you think? but will add that wrapped... Ofc if you think otherwise strongly happy to change, i will not die on this hill 😅